fe/fe-core/src/main/java/com/starrocks/sql/ast/UserVariable.java

```diff
@@ -38,12 +38,15 @@
 
 import java.nio.ByteBuffer;
 import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
 import java.util.List;
 
 public class UserVariable extends SetListItem {
     private final String variable;
     private Expr unevaluatedExpression;
     private Expr evaluatedExpression;
+    private List<String> userVariableDependencyWithoutFind = new ArrayList<>();
```

Comment:
what does this variable stand for?
Does it mean the variables that this variable depends on?
But I'm confused by the `WithoutFind`, could you explain more about it?

Reply:
It is a little bit confusing. How about `prevUserVariableDependency`? I think user variable only reference previous user variable need be re-analyze.